instance/templates/instance/ansible/swift.yml

```diff
@@ -0,0 +1,6 @@
+EDXAPP_SWIFT_AUTH_VERSION: '2'
+EDXAPP_SWIFT_USERNAME: '{{ user }}'
+EDXAPP_SWIFT_KEY: '{{ password }}'
+EDXAPP_SWIFT_TENANT_NAME: '{{ tenant }}'
+EDXAPP_SWIFT_AUTH_URL: '{{ auth_url }}'
+EDXAPP_SWIFT_REGION_NAME: '{{ region }}'
```

Comment:
To be able to use Swift containers with https://github.com/edx/edx-platform/pull/11286, we will have to set `EDXAPP_DEFAULT_FILE_STORAGE` to `'swift.storage.SwiftStorage'`. I suppose this would be the right place to do that.
We will also have to set `EDXAPP_SETTINGS` to `opencraft` somewhere - but I think we can do that in [ansible/vars.yml](https://github.com/open-craft/opencraft/blob/smarnach/swift/instance/templates/instance/ansible/vars.yml).